specs/004-npm-to-bun/spec.md

```
@@ -0,0 +1,60 @@
+
+## リスク
+
+| リスク | 対策 |
+|-------|------|
+| Bun と Next.js の互換性問題 | Next.js の実行には Node.js を併用（oven/bun は Node.js も含む） |
+| CI での Bun バージョン差異 | Dockerfile と CI で同じバージョン（1.3）を使用 |
+
+## 関連ドキュメント
+
+- [plan.md](plan.md) - 実装計画
+- [tasks.md](tasks.md) - タスク一覧
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)